### PR TITLE
Add projects from Notion (8/3)

### PR DESCRIPTION
**Pull request by thebeyondr**
## Add projects, update project documentation

**What's new:**
- ZKOPRU + banner
- Interep + banner
- COCO + banner
- ECIP
- Perpetual Towers of Tau
- ZK Kit + banner

### Next steps:
The projects should be tagged

### CODE REVIEW

**Review by ChialiT — APPROVED** [review]

File: data/projects/coco.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        {
+            label: 'Frontend',
+            url: 'https://github.com/Janmajayamall/coco-frontend'
+        },
+        {
+            label: 'Frontend (General)',
+            url: 'https://github.com/Janmajayamall/coco-frontend-general'
+        }],
+    },
+}
```
Comment (ChialiT) [inline data/projects/coco.ts:28]: can add "prediction market" & "scaling" in the keyword

File: data/projects/interep.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        github: 'https://github.com/interep-project',
+        youtube: 'https://www.youtube.com/watch?v=dYKgHkb_aqk',
+    },
+    extraLinks: {
+        learn: [{
+            label: 'Interep: An on-ramp for reputation',
+            url: 'https://mirror.xyz/privacy-scaling-explorations.eth/w7zCHj0xoxIfhoJIxI-ZeYIXwvNatP1t4w0TsqSIBe4'
+        }],
+    },
+}
```
Comment (ChialiT) [inline data/projects/interep.ts:21]: "social" "reputation" in the keyword

File: data/projects/powers-of-tau.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    github: 'https://github.com/privacy-scaling-explorations/perpetualpowersoftau',
+    website: 'https://perpetualpowersoftau.com/'
+  },
+  extraLinks: {
+    learn: [{
+      label: 'Announcing the Perpetual Powers of Tau Ceremony',
+      url: 'https://medium.com/coinmonks/announcing-the-perpetual-powers-of-tau-ceremony-to-benefit-all-zk-snark-projects-c3da86af8377'
+    }],
+  },
+}
```
Comment (ChialiT) [inline data/projects/powers-of-tau.ts:20]: "scaling"

**Review by ChialiT — COMMENTED** [review]
suggested few keywords

**Review by kalidiagne — APPROVED** [review]

**Pull request merged**